teamA/leeHyeonTae/models/postDao.js

```diff
@@ -0,0 +1,64 @@
+const { DataSource } = require('typeorm');
+
+const appDataSource = new DataSource({
+    type: process.env.TYPEORM_CONNECTION,
+    host: process.env.TYPEORM_HOST,
+    port: process.env.TYPEORM_PORT,
+    username: process.env.TYPEORM_USERNAME,
+    password: process.env.TYPEORM_PASSWORD,
+    database: process.env.TYPEORM_DATABASE
+});
+
+appDataSource.initialize()
+    .then( () => {
+        console.log('Data Source has been initialzed');
+    })
+    .catch( (err) => {
+        console.error('error occured during data source initalization',err);
+        appDataSource.destroy();
+    })
+
+    const createPost = async ( title, content, url_image, user_id) => {
+        try{
+            return await appDataSource.query(
+                `INSERT INTO posts(
+                    title,
+                    content,
+                    url_image,
+                    user_id
+                ) VALUES ( ?,?,?,? )`
+                , [ title, content, url_image, user_id]
+            )
+        }
+        catch (err){
+            const error = new Error('invalid data input check your Dao');
+            err.statusCode = 500;
+            throw error;
+        }
+    }
+
+    const searchPost = async () => {
+        try{
+            return await appDataSource.query(
+                `SELECT 
+                        posts.id as postingId,
+                        posts.content as postingContent,
+                        posts.url_image as postingImageUrl,
+                        posts.user_id as userId,
+                        users.profile_image as userProfileImage
+                        FROM (posts, users)
+                        WHERE users.id=posts.user_id;
+
+                `
```

Comment:
- 해당 쿼리문 SQL indentation을 맞춰서 다시 작성해 주세요 !
```suggestion
        try{
            return await appDataSource.query(
                `SELECT 
                        posts.id as postingId,
                        posts.content as postingContent,
                        posts.url_image as postingImageUrl,
                        posts.user_id as userId,
                        users.profile_image as userProfileImage
                    FROM (posts, users)
                    WHERE users.id=posts.user_id;`
```